### PR TITLE
Added GetNotificationHub method

### PR DESCRIPTION
**Pull request by iTOYS**

### CODE REVIEW

File: src/Microsoft.Azure.NotificationHubs/NamespaceManager.cs [hunk 1]
```diff
@@ -243,47 +243,47 @@ public async Task<NotificationHubDescription> CreateNotificationHubAsync(Notific
             };
 
             var token = _settings.TokenProvider.GetToken(uriBuilder.Uri.ToString());
+            using (var client = new HttpClient())
```
Comment (mpodwysocki) [inline src/Microsoft.Azure.NotificationHubs/NamespaceManager.cs]: If the HttpClient is out here using the IDisposable pattern, could it be disposed before the execute on a retry policy is done?

Comment (iTOYS) [reply src/Microsoft.Azure.NotificationHubs/NamespaceManager.cs]: No, it will be disposed only when unhandled exception happen or on the end of the method

Comment (mpodwysocki) [reply src/Microsoft.Azure.NotificationHubs/NamespaceManager.cs]: Trying to compare between the inconsistent styles between here and [here](https://github.com/Azure/azure-notificationhubs-dotnet/pull/59/files#diff-c6fdacc734cc199a2498fe9282251c8fR256)

File: src/Microsoft.Azure.NotificationHubs/NamespaceManager.cs [hunk 2]
```diff
@@ -325,6 +374,9 @@ public Task<IEnumerable<NotificationHubDescription>> GetNotificationHubsAsync()
             throw new NotImplementedException();
         }
 
+        /// <summary>
```
Comment (mpodwysocki) [inline src/Microsoft.Azure.NotificationHubs/NamespaceManager.cs]: If we are going to add XML comments, add the complete XML parts including the parameter

File: src/Microsoft.Azure.NotificationHubs/NamespaceManager.cs (hunk 1 above)
Comment (iTOYS) [reply src/Microsoft.Azure.NotificationHubs/NamespaceManager.cs]: I fixed inconsistent styles in this PR. Artem's PR is target to my PR

**Pull request merged**